PORTAL/jobs/_utils.py

```diff
@@ -1375,7 +1380,8 @@ def timestamp(self):
         try:
             return self._timestamp
         except AttributeError:
-            self._timestamp = get_utc_timestamp(self.header.created)
+            # TODO: get_utc_timestamp is undefined
+            self._timestamp = get_utc_timestamp(self.header.created)  # noqa
```

Comment:
I'm pretty sure it should be:

```suggestion
            self._timestamp = get_utc_datetime(self.header.created)
```